components/TestResult/assertions.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+export default function assertions(mountStory: (options?: any) => void): void {
+  beforeEach(() => {
+    mountStory()
+  })
+
+  afterEach(() => {
+    cy.percySnapshot()
+  })
+
+  describe('@container', { viewportWidth: 1024, viewportHeight: 400 }, () => {
+    it('renders all sizes', () => {
+      cy.viewport(320, 400).log('@xs')
+
+      cy.viewport(384, 400).log('@sm')
+
+      cy.viewport(448, 400).log('@md')
+
+      cy.viewport(512, 400).log('@lg')
+
+      cy.viewport(576, 400).log('@xl')
+
+      cy.viewport(672, 400).log('@2xl')
+
+      cy.viewport(768, 400).log('@3xl')
+
+      cy.viewport(896, 400).log('@4xl')
+
+      cy.viewport(1024, 400).log('@5xl')
+    })
+
+    it('expand when groups are present', () => {
+      cy.get('[data-cy="cd-tr-container"]').eq(-1).as('container')
```

Comment:
@elevatebart I could use your help on this one. It feels a bit dirty and brittle to use `eq()` here instead of passing through a property that better identifies each component. I was struggling to get that to work properly though locally when trying to use other properties like `id` or even a new data attribute like `data-cy-id` or something like that.